MapboxCoreNavigation/String.swift

```diff
@@ -52,4 +52,8 @@ extension String {
     var isUppercased: Bool {
         return self == uppercased() && self != lowercased()
     }
+    
+    var isInt: Bool {
+        return Int(self) != nil
```

Comment:
I think this will miss route numbers like [`NY 120A`](https://en.wikipedia.org/wiki/New_York_State_Route_120A), in which `120A` should be wrapped in `interpret-as="address"` in order to say “one-twenty a” instead of “one hundred twenty a”. How about wrapping a word in `address` if it contains a digit?